.gitlab-ci.yml

```diff
@@ -111,6 +111,8 @@ prepare-rhel-internal-runners:
     - aws/rhel-8.10-ga-x86_64
     - aws/rhel-9.4-ga-x86_64
     # - aws/rhel-9.5-ga-x86_64
+    - aws/rhel-9.6-nightly-x86_64
```

Comment:
I know it's a bit messy, but we have nightly builds separated in two jobs inside this same file:
- image-builder-aws-nightly-compose
- image-builder-azure-nightly-compose

So, nightly runners are already added there and we are OK:
.rhel_runners_x86_64_internal_nightlies
.rhel_runners_aarch64_internal_nightlies

Reply:
so line #114 is not needed then?